src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/AccessToken/config_body_default.yml

```diff
@@ -0,0 +1,32 @@
+imports:
+    - { resource: ./../config/framework.yml }
+
+framework:
+    http_method_override: false
+    serializer: ~
+
+security:
+    enable_authenticator_manager: true
+    password_hashers:
+        Symfony\Component\Security\Core\User\InMemoryUser: plaintext
+
+    providers:
+        in_memory:
+            memory:
+                users:
+                    dunglas: { password: foo, roles: [ROLE_USER] }
+
+    firewalls:
+        main:
+            pattern: ^/
+            access_token:
+                realm: 'My API'
+                token_handler: access_token.access_token_handler
+                token_extractors: 'security.access_token_extractor.request_body'
```

Comment:
Could we give extractors some short names that can be referenced from the config i.e. `["query_string" "request_body", "header"]`? Maybe no real extension point but just hardcoded at the DI level, for built-in extractors?

Reply:
Nice idea. I added those aliases in the factory and use them for functional tests